app/Listeners/UserVerificationStatusChangeNotification.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Events\UserVerificationStatusChange;
+use App\Notification\Notifier;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class UserVerificationStatusChangeNotification implements ShouldQueue
+{
+    /**
+     * Create the event listener.
+     *
+     * @return void
+     */
+    public function __construct(Notifier $notifier)
+    {
+        $this->notifier = $notifier;
+    }
+
+    /**
+     * Handle the event.
+     *
+     * @param  UserVerificationStatusChange  $event
+     * @return void
+     */
+    public function handle(UserVerificationStatusChange $event)
+    {
+        $text = "Your account verification status has changed from {$oldValue} to {$newValue}.";
```

Comment:
I'm getting `Undefined variable: oldValue` here. I see that in implements the above class though, should it be using `$this->oldValue`? Is this a PHP version issue?